### PR TITLE
feat: add noheader option

### PR DESCRIPTION
**Pull request by qoomon**
fixes: #46

### CODE REVIEW

**Comment by qoomon** [comment]
Anything missing?

File: src/update-readme.ts [hunk 1]
```diff
@@ -49,7 +54,9 @@ export function generateReadme(
     content =
       content.slice(0, start) +
       `${startString}\n` +
-      '<h4><a href="https://github.com/my-badges/my-badges">My Badges</a></h4>\n\n' +
+      (!omitheader
```
Comment (antongolub) [inline src/update-readme.ts:57]: Let's use var here:

```js 
const headerHtml = omitheader ? '' : '<...>'
```

Comment (antongolub) [reply src/update-readme.ts:57]: `pick / omit` and `omitheader`. Maybe just `noheader`?

CC @antonmedv

Comment (antonmedv) [reply src/update-readme.ts:57]: noheader better

**Comment by qoomon** [comment]
Done

**Comment by antongolub** [comment]
The option should be mentioned in README.md

**Comment by qoomon** [comment]
Done

**Comment by antonmedv** [comment]
image

**Comment by antonmedv** [comment]
Lets remove header all together!

**Pull request closed without merging**